timelock-impl/src/main/java/com/palantir/atlasdb/timelock/paxos/PaxosQuorumCheckingCoalescingFunction.java

```diff
@@ -111,7 +112,7 @@ private PaxosQuorumCheckingCoalescingFunction(
     public static <REQ, RESP extends PaxosResponse, SERVICE, F extends CoalescingRequestFunction<REQ, RESP>>
     PaxosQuorumCheckingCoalescingFunction<REQ, RESP, F> wrapWithRemotes(
             List<SERVICE> services,
-            Map<SERVICE, ExecutorService> executors,
+            Map<SERVICE, CheckedRejectionExecutorService> executors,
             int quorumSize,
             Function<SERVICE, F> functionFactory) {
         List<FunctionAndExecutor<F>> functionsAndExecutors = KeyedStream.of(services)
```

Comment:
Where is this used?

Reply:
It's used inside `wrap()`. I think you might be referencing the private constructor on line 68 though, which is unused? I've removed that.

Reply:
Ah, I guess it's not. Well spotted. I'll remove this.